calico-enterprise/getting-started/install-on-clusters/openshift/installation.mdx

```diff
@@ -2,9 +2,12 @@
 description: Install Calico Enterprise on an OpenShift 4 cluster.
 ---
 
+import InstallOpenShift from '@site/calico-enterprise/_includes/components/InstallOpenShift';
+import InstallOpenshiftBeforeYouBegin from '@site/calico-enterprise/_includes/components/InstallOpenshiftBeforeYouBegin';
+
 # Install Calico Enterprise on OpenShift
 
-import InstallOpenShiftManifests from '@site/calico-enterprise/_includes/components/InstallOpenShiftManifests.js';
+import InstallOpenShiftManifests from '@site/calico-enterprise/_includes/components/InstallOpenShiftManifests';
```

Comment:
Could you group all imports together? Thanks 🙏

Reply:
Nice spot, thanks! Missed this one.